bin/ci

```diff
@@ -4,3 +4,4 @@ set -e
 
 bin/rspec spec
 bin/quality
+bin/rake jasmine:ci
```

Comment:
thanks, fixed in https://github.com/fs/rails-base/commit/037da942558f8dec2ce2a81bc86f2c5d1961b856